ARKit+CoreLocation/POIViewController.swift

```diff
@@ -362,11 +365,20 @@ extension POIViewController {
 extension POIViewController: LNTouchDelegate {
 
     func annotationNodeTouched(node: AnnotationNode) {
-        print("AnnotationNode touched \(node)")
+		if let node = node.parent as? LocationNode {
+			let coords = "\(node.location.coordinate.latitude.short) \(node.location.coordinate.longitude.short)"
```

Comment:
Could we have degree symbols, and "m" for the altitude? I didn't understand what I was seeing at first, thought they might have been x/y/z world coordinates.

Reply:
Done.  I thought about adding additional text such as full word "meters", or "N,E,S,W" for the coords, but skipped that to be respectful for narrow devices.

![image](https://user-images.githubusercontent.com/12265773/69469593-4a6a0780-0d57-11ea-9258-9c580c8e6057.png)

Reply:
There's lots of good stuff at https://github.com/FormatterKit/FormatterKit for angles. Not an appropriate dependency for a library demo, but I use several of those formatters for production.